Mathlib/Logic/Function/Basic.lean

```diff
@@ -7,6 +7,7 @@ import Std.Classes.SetNotation
 import Std.Tactic.Lint.Basic
 import Mathlib.Data.Option.Defs
 import Mathlib.Logic.Basic
+import Mathlib.Logic.Nonempty
```

Comment:
Spurious?

Reply:
No, that's purposeful, because that is present in mathlib3, and this allows transitively lemmas like `nonempty_Prop` work in proofs of `is_empty_Prop`.